google/cloud/storage/_helpers.py

```diff
@@ -599,7 +600,7 @@ def _get_default_headers(
         "Accept": "application/json",
         "Accept-Encoding": "gzip, deflate",
         "User-Agent": user_agent,
-        "x-goog-api-client": user_agent,
+        "x-goog-api-client": user_agent + " gccl-invocation-id/" + uuid4(),
```

Comment:
IIUC this uuid4() call will only occur once*;

Reply:
From my understanding, that is true. Tests will confirm.